charts/loki-simple-scalable/values.yaml

```diff
@@ -173,6 +173,17 @@ prometheusRule:
 write:
   # -- Number of replicas for the write
   replicas: 1
+  autoscaling:
+    # -- Enable autoscaling for the write
+    enabled: false
+    # -- Minimum autoscaling replicas for the write
+    minReplicas: 1
+    # -- Maximum autoscaling replicas for the write
+    maxReplicas: 3
```

Comment:
and we could then bump `maxReplicas` to 6?

Reply:
I support setting it to maxReplicas = 3 initially so no immediately scaling for testing. The user could later change maxReplicas value once they feel confident / full tested to handle scaling.